ext/pdo/pdo_dbh.stub.php

```diff
@@ -49,5 +49,5 @@ public function rollBack() {}
      * @param array|int $value
      * @return bool
      */
-    public function setAttribute(int $attribute, $value) {}
+    public function setAttribute(int $name, $value) {}
```

Comment:
why `$name`? It seems to be a weird name for an `int` parameter. I would be ok with just `$attribute`.

Reply:
Good point; I didn't look at the type, just at the odd redundancy of "attribute".  I'll revert.